### PR TITLE
Add Documentation for IRSA cross account

### PR DESCRIPTION
**Pull request by paurosello**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

**Comment by paurosello** [comment]
Do you think this is clear or we need to add some diagrams with actual AWS account examples?

File: src/content/advanced/iam-roles-for-service-accounts/index.md [hunk 1]
```diff
@@ -127,6 +127,47 @@ metadata:
...
+                    "s3.CLUSTER_REGION.amazonaws.com/CLUSTER_AWS_ACCOUNT-g8s-CLUSTER_ID-oidc-pod-identity:aud": "sts.amazonaws.com"
+                }
+            }
+        }
+    ]
+}
+```
+
+You need to fill real values for these placeholders:
+* `CLUSTER_AWS_ACCOUNT` - aws account id, where is the cluster running
```
Comment (njuettner) [inline src/content/advanced/iam-roles-for-service-accounts/index.md:165]:
```suggestion
* `CLUSTER_AWS_ACCOUNT` - AWS account id, where is the cluster running
```

File: src/content/advanced/iam-roles-for-service-accounts/index.md [hunk 2]
```diff
@@ -127,6 +127,47 @@ metadata:
...
 ```
 
+## Cross Account Roles
+
+In order to assume roles cross AWS accounts you will need to create an AWS Identity Provider (OpenID Connect) in the AWS account where the role is located.
+
+![Creating AWS Identity Provider](identity-provider.png)
+
+The parameters to create the Identity Provider are:
+- Set Provider URL to the `OpenID Connect provider URL` of the kubernetes where you want to assumer the role from and looks like `https://s3.REGION.amazonaws.com/AWS_ACCOUNT-g8s-CLUSTER_ID-oidc-pod-identity`. Once the URL is set-up, click the `Get thumbprint` to import the OIDC keys.
```
[the remark this answers is not in the document]
Comment (paurosello) [reply src/content/advanced/iam-roles-for-service-accounts/index.md:137]: yes

File: src/content/advanced/iam-roles-for-service-accounts/index.md [hunk 3]
```diff
@@ -127,6 +127,47 @@ metadata:
...
+- Set the audience to sts.amazonaws.com
```
[the remark this answers is not in the document]
Comment (paurosello) [reply src/content/advanced/iam-roles-for-service-accounts/index.md:138]: Done

File: src/content/advanced/iam-roles-for-service-accounts/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (njuettner) [reply src/content/advanced/iam-roles-for-service-accounts/index.md:137]: Could we rephrase it a bit 🥺?

**Pull request merged**